### ISSUE TITLE
aws-sso-util broken by incompatible botocore release

### ISSUE THREAD

**Issue by kapet** [benkehoe/aws-sso-util#70]
with pull request [2743](https://github.com/boto/botocore/pull/2743) having been merged into botocore, there is a problem now:

aws-sso-util assumes it needs to return only the accessToken from the token loader:
https://github.com/benkehoe/aws-sso-util/blob/master/lib/aws_sso_lib/sso.py#L95

but now botocore expects to be given a dictionary by the token loader, and then extract the accessToken itself:
https://github.com/boto/botocore/blob/develop/botocore/credentials.py#L2167

The result of this is a "string indices must be integers" exception.

**Comment by benkehoe** [comment]
The perils of relying on internal botocore details. Thanks for this. This appears to be in [botocore version 1.27.60](https://github.com/boto/botocore/blob/develop/CHANGELOG.rst#12760) (boto3 version 1.24.60). I'll take a look.

**Comment by benkehoe** [comment]
This appears to only affect the `get_credentials()` function. I will fix it, but is there a reason you're using that instead of `get_boto3_session()`?

**Comment by kapet** [comment]
Thanks Ben! I'm calling `aws-sso-util credential-process`, which seems to always call `get_credentials`:
https://github.com/benkehoe/aws-sso-util/blob/master/cli/src/aws_sso_util/credential_process.py#L159

**Comment by rauchrob** [comment]
Hi folks, I just ran into this issue. Is there a quick workaround for the meantime?

**Comment by kapet** [comment]
This workaround worked for us:
`~/.local/pipx/venvs/aws-sso-util/bin/python -m pip install botocore==1.27.58`
It's a slightly older version of botocore from before the incompatible code change.

**Comment by benkehoe** [comment]
The newer version of botocore also doesn't support 3.6, I've been trying to figure out how to support both, I think I need to just deprecate 3.6 as well

**Comment by rauchrob** [comment]
> This workaround worked for us: `~/.local/pipx/venvs/aws-sso-util/bin/python -m pip install botocore==1.27.58` It's a slightly older version of botocore from before the incompatible code change.

I just tried to verify this workaround after a fresh installation of `aws-sso-util` via`pipx` and found that I also needed to downgrade the `boto3` package to avoid dependency conflicts. Thus, the following worked for me:
```
~/.local/pipx/venvs/aws-sso-util/bin/python -m pip install botocore==1.27.58 boto3==1.24.58
```

**Comment by benkehoe** [comment]
This should be fixed in [aws-sso-lib v1.12](https://github.com/benkehoe/aws-sso-util/releases/tag/lib-v1.12) and [aws-sso-util v4.29](https://github.com/benkehoe/aws-sso-util/releases/tag/cli-v4.29).
